### PR TITLE
shelltestrunner 1.9 (new formula)

### PR DESCRIPTION
**Pull request by taavi-valjaots**
Audit is failing with "GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)". 

Some good words about the project:
This is a tool that can be used for testing other command-line tools. I'm not the author but I have used it for couple of years (and contributed a little). It's useful, stable and is included to cabal and it's older version is available on Debian / Ubuntu. Unfortunately for OSX it's not packaged and must be built but most people doesn't have haskell installed - so its a ugly and long process.

- [ok] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ok] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ok] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ok] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/shelltestrunner.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require "language/haskell"
+
+class Shelltestrunner < Formula
+  include Language::Haskell::Cabal
+
+  desc "Portable command-line tool for testing command-line programs"
+  homepage "https://github.com/simonmichael/shelltestrunner"
+  url "https://github.com/simonmichael/shelltestrunner/archive/1.9.tar.gz"
```
Comment (ilovezfs) [inline Formula/shelltestrunner.rb]: let's use the Hackage tarball

**Review by ilovezfs — CHANGES_REQUESTED** [review]

**Comment by taavi-valjaots** [comment]
Fixes implemented.

I'm a little bit confused how are the bottles built? I know that `brew bottle formula` builds the bottle but how to make it available? Should I do something or is it done by brew maintainers?

**Comment by woodruffw** [comment]
> I'm a little bit confused how are the bottles built? I know that brew bottle formula builds the bottle but how to make it available? Should I do something or is it done by brew maintainers?

Our CI and bots take care of the bottle-building and formula-patching process -- you don't need to do anything with respect to that 🙂.

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks for your first contribution to Homebrew, @taavi-valjaots! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**